src/Http/Http/perf/Microbenchmarks/ValidatableTypesBenchmark.cs

```diff
@@ -244,6 +244,7 @@ public IEnumerable<ValidationResult> Validate(ValidationContext validationContex
 
     private class MockValidatableTypeInfo(Type type, ValidatablePropertyInfo[] members) : ValidatableTypeInfo(type, members)
     {
+        protected override ValidationAttribute[] GetValidationAttributes() => [];
```

Comment:
Since the ValidatableTypeInfo is marked as experimental, we're OK to make this change without API review. Also, it follows the same patter as the ValidatablePropertyInfo that was already code reviewed so we should be good there anyways.